okhttp/src/test/java/io/grpc/okhttp/OkHttpProtocolNegotiatorTest.java

```diff
@@ -110,7 +112,9 @@ public void negotiatorNotNull() {
 
   @Test
   public void negotiate_handshakeFails() throws IOException {
+    SSLParameters parameters = mock(SSLParameters.class);
```

Comment:
Why not `new SSLParameters()`? Please use real objects when easily possible.

Reply:
Done